dev_guide/builds.adoc

```diff
@@ -11,11 +11,76 @@ toc::[]
 
 == Overview
 A link:../architecture/core_objects/builds.html[build] is a process of creating
-runnable images to be used on OpenShift. There are three types of builds:
+runnable images to be used on OpenShift. There are three build strategies:
 
-- Docker build
-- STI build
-- custom build
+- link:../architecture/core_objects/builds.html#sti-build[Source-To-Image]
+- link:../architecture/core_objects/builds.html#docker-build[Docker]
+- link:../architecture/core_objects/builds.html#custom-build[Custom]
+
+== Defining a buildConfig
+
+A `*buildConfig*` describes a single build definition and a
```

Comment:
it is not about json but REST endpoint name
On Jun 9, 2015 09:50, "Rodolfo Carvalho" <notifications@github.com> wrote:

> In dev_guide/builds.adoc
> <https://github.com/openshift/openshift-docs/pull/444#discussion_r31989407>
> :
>
> >
> > -- Docker build
> > -- STI build
> > -- custom build
> > +- link:../architecture/core_objects/builds.html#sti-build[Source-To-Image]
> > +- link:../architecture/core_objects/builds.html#docker-build[Docker]
> > +- link:../architecture/core_objects/builds.html#custom-build[Custom]
> > +
> > +== Defining a buildConfig
> > +
> > +A `*buildConfig*` describes a single build definition and a
>
> The JSON reads "kind": "BuildConfig", ..., I also think it should be
> BuildConfig and DeploymentConfig.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/openshift-docs/pull/444/files#r31989407>.
>